### ISSUE TITLE
Old plugins

### ISSUE THREAD

**Issue by lucpretti** [okfn/website#896]
Here's a list of actions to clean the list of the available plugins:

(Basically, delete everything keeping only `Text` and `Image`)

| Group   | Plugin Name          | Action |
|---------|----------------------|--------|
| Generic | Article list item    | Delete |
|         | File                 | Delete |
|         | Folder               | Delete |
|         | Image                | **Keep**   |
|         | Link                 | Delete |
|         | Quote                | Delete |
|         | Text                 | **Keep**   |
|         | Video                | Delete |
| OKF     | Featured News        | Delete |
|         | Featured Project     | Delete |
|         | Featured Theme       | Delete |
|         | Latest Blogposts     | Delete |
|         | Network Groups Flags | Delete |
|         | Press Mentions       | Delete |
|         | Press Releases       | Delete |
|         | Press Resources      | Delete |
|         | Project List         | Delete |
|         | Theme List           | Delete |
|         | Working Groups       | Delete |

**Comment by pdelboca** [comment]
Working on this I found out that we need to decide what to do with the models. Cleaning the plugin code will not create a migration file since the plugin does not still exist.

Reading the docs I found that `django-cms` provides some commands to delete orphan plugins and clean data:
 - https://docs.django-cms.org/en/latest/reference/cli.html#cms-delete-orphaned-plugins
 - https://docs.django-cms.org/en/latest/reference/cli.html#cms-uninstall

**Comment by pdelboca** [comment]
Created a PR for some of this plugins: #919.

I think that a safe approach could be to uninstall all the plugins in next.okfn.org first by running `python manage.py cms uninstall plugins [...]` (and same with apphooks)

It will also give us less headaches since cleaning a lot of orphaned plugins could easily cause errors due to references in ForeignKeys (See notes in #919 )

**Comment by amercader** [comment]
Sounds good @pdelboca 
Let's try it on next and see how it goes. As you say we can always restore the production database

**Comment by pdelboca** [comment]
Today I have been testing several scenarios. Here are some things to consider:
 - tunning `python manage.py cms uninstall plugin <plugin_name>` will drop the data but not the table.
 - 'cms uninstall` will work only if the cms_plugin is in the `settings.py`
 - After uninstalling we can safely delete the code
 - Removing a `foundation.<plugin>` from `settings.py` and running `delete-orphaned-plugins` will erase only the data in the `cms_cmsplugin` table but not in the table of the plugin itself

**Comment by pdelboca** [comment]
@lucpretti I have removed `ArticleListItem`, and all the `Press` related plugins. Do you have a way to test it in staging?

**Comment by pdelboca** [comment]
Also @lucpretti I see in the list that Featured News needs to be deleted. 

However I see some things in the site related to it. Can you confirm?

![image](https://github.com/okfn/website/assets/6672339/7e5c1a42-5c38-42a3-95e0-26b5fc7092ac)

**Comment by lucpretti** [comment]
> @lucpretti I have removed `ArticleListItem`, and all the `Press` related plugins. Do you have a way to test it in staging?

I can still see the `ArticleListItem` listed under the `Generic` plugins group. I tried to use it and yes, I'm still able to do it.
![Screenshot 2023-07-21 at 11 41 52](https://github.com/okfn/website/assets/8523847/5c3be473-e3aa-4bbc-b9c0-ddce54a54834)

The Press related and others were successfully deleted from the `OKF` group. See below:

![Screenshot 2023-07-21 at 11 42 03](https://github.com/okfn/website/assets/8523847/46c27b9d-a6f4-43f6-9098-59f25d03cd44)

**Comment by lucpretti** [comment]
> Also @lucpretti I see in the list that Featured News needs to be deleted.
> 
> However I see some things in the site related to it. Can you confirm?
> 
> ![image](https://user-images.githubusercontent.com/6672339/255137367-7e5c1a42-5c38-42a3-95e0-26b5fc7092ac.png)

You are right. Before deleting Featured News, we need to make sure #902 is done.

**Comment by pdelboca** [comment]
@lucpretti I'm moving this to out for review since Staging should already have all the required cleanup.

In the latest, I uninstalled the generic plugins: https://github.com/okfn/website/pull/955/files

Note that I'm removing the content, not the plugin itself since we need to do a two step process (Content, then Code)

The only thing that will remind to completely close is to remove the `auldryn` plugins code after we merge this into PRD.

**Comment by lucpretti** [comment]
Thank you!

I feel you know what you're doing, so I'm not concerned :) But I'm not really clear about the difference between Deleted and Uninstalled though. Some of the Uninstalled are still available on the plugins menu (like `File`, `Folder`, etc):

![Screenshot 2023-08-22 at 10 58 08](https://github.com/okfn/website/assets/8523847/e76e51d2-951b-4c8c-a810-d1906523ae31)

The ideal situation would be really removing them from the menu, leaving only the plugins styled for v2.
